src/main/java/reposense/util/FileUtil.java

```diff
@@ -122,8 +127,70 @@ public static void copyDirectoryFiles(Path src, Path dest) throws IOException {
         }
     }
 
+    /**
+     * Generates a list of Path contained in the given {@code directoryName} directory.
+     */
+    public static List<Path> listJsonFiles(String directoryName) {
+        Path directory = Paths.get(directoryName);
+        List<Path> allJsonFiles = null;
+        try {
+            allJsonFiles = Files.walk(directory)
+                    .filter(p -> p.toString().endsWith(".json"))
+                    .distinct()
+                    .collect(Collectors.toList());
+        } catch (IOException ioe) {
+            logger.log(Level.SEVERE, ioe.getMessage(), ioe);
+        }
+
+        return allJsonFiles;
+    }
+
+    /**
+     * Zips all the JSON files contained in the {@generatedFolder} directory.
+     * Creates the zip folder in the {@code directoryPath}
+     */
+    public static void zip(String directoryPath, String generatedFolder) {
+
+        String zipFileName = "archiveJSON.zip";
```

Comment:
As this method is in ```FileUtil```, this variable should not be hard coded. Pass it as a parameters, to make this method reusable.